app/services/channel_queues/response_retriever.rb

```diff
@@ -18,25 +18,32 @@ def self.retrieve(original_request_text, channel_id, channel_name, user_id, user
         json_result[:text] = channel_queue.members_string
       when 'join'
         user = find_user(user_id, user_name)
-        json_result[:text] = "#{user.slack_user_name} already in queue for #{channel_queue.slack_channel_name}." if ChannelQueueMembership.where(user: user, channel_queue: channel_queue).exists?
-        return json_result if ChannelQueueMembership.where(user: user, channel_queue: channel_queue).exists?
+        if ChannelQueueMembership.where(user: user, channel_queue: channel_queue).exists?
+          json_result[:text] = "#{user.slack_user_name} already in queue for #{channel_queue.slack_channel_name}."
+          json_result[:response_type] = 'ephemeral'
+          return json_result
+        end
 
         ChannelQueueMembership.create!(user: user, channel_queue: channel_queue)
         json_result[:text] = "#{user.slack_user_name} joined queue for #{channel_queue.slack_channel_name}."
```

Comment:
Should we display the list when someone joins the queue?

Reply:
Yup! I have it as a separate story that I'll work on tomorrow